README.md

```diff
@@ -56,7 +56,7 @@ git clone https://github.com/ros-industrial-consortium/bezier.git
 
 **Launch**
 ```bash
-roslaunch bezier_application bezier_application_r1000ia.launch meshname:=plane/plane.ply
+roslaunch bezier_application bezier_application_r1000ia.launch meshname:=plane/plane.ply grinding:=full
 ```
 
 In this example, `bezier_application` will be launched with `oriented_complicated_mesh.ply` as input mesh and `oriented_complicated_mesh_defect.ply` as default mesh.
```

Comment:
- Rename `grinding` to `grinding_surfacing_mode`.
- Default value should be `off`, `on` triggers the mode

Add a comment stating what is grinding